components/builder-depot/src/doctor.rs

```diff
@@ -154,7 +153,6 @@ impl<'a> Doctor<'a> {
 
     fn run(mut self) -> Result<Report> {
         try!(self.init_fs());
-        try!(self.truncate_datastore(&self.depot.datastore));
```

Comment:
I didn't add that in because with the sharding it's more involved and I'm really not sure we even want to keep the doctor "as is". I'm thinking we change it to NOT truncate but to be able to scan a storage system and ADD packages that it does not already have. That is super simple to change and was thinking it would be a separate PR. Thoughts?